src/adaptors/clearpool/index.js

```diff
@@ -2,70 +2,37 @@
 // pnpm i jest -g
 // npm run test --adapter=clearpool
 
+const { request, gql } = require('graphql-request');
 const utils = require('../utils');
 
 const poolsFunction = async () => {
-  const networks = {
-    1: 'Ethereum',
-    10: 'Optimism',
-    137: 'Polygon',
-    1101: 'Polygon zkEVM',
-    5000: 'Mantle',
-  };
-  /** CPOOL token address for each chain */
-  const rewardTokens = {
-    1: '0x66761fa41377003622aee3c7675fc7b5c1c2fac5',
-    10: '0xc3630b805F10E91c2de084Ac26C66bCD91F3D3fE',
-    137: '0xb08b3603C5F2629eF83510E6049eDEeFdc3A2D91',
-    1101: '0xc3630b805F10E91c2de084Ac26C66bCD91F3D3fE',
-    5000: '0x0c8927de225Bb1eD9DB05aA7d641E434B95279d8',
-  };
-  /** USDC token address for each chain */
-  const underlyingTokensUSDC = {
-    1: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
-    10: '0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85',
-    137: '0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0',
-    1101: '0xA8CE8aee21bC2A48a5EF670afCc9274C7bbbC035',
-    5000: '0x09Bc4E0D864854c6aFB6eB9A9cdF58aC190D0dF9',
-  };
-  /** USDT token address for each chain */
-  const underlyingTokensUSDT = {
-    1: '0xdAC17F958D2ee523a2206206994597C13D831ec7',
-    10: '0x94b008aA00579c1307B0EF2c499aD98a8ce58e58',
-    137: '0xc2132D05D31c914a87C6611C10748AEb04B58e8F',
-    1101: '0x1E4a5963aBFD975d8c9021ce480b42188849D41d',
-    5000: '0x201EBa5CC46D216Ce6DC03F6a759e8E766e956aE',
-  };
-  const dataPools = await utils.getData('https://clearpool.finance/api/pools');
-  let pools = [];
 
-  for (const chainId in dataPools) {
-    for (const pool of dataPools[chainId]) {
-      const chainName = utils.formatChain(networks[chainId]);
-
-      pools.push({
-        pool: `${pool.address}-${chainName}`.toLowerCase(),
-        chain: chainName,
```

Comment:
no sure here is we will preserve the same format as it was, please check

Reply:
@georgeciubotaru its same format.  I didn't used `poolAddress` & `protocol` fields from the API. all good from my end.
below are the sample pools generated after running the test cases.
- the only difference is in the existing we used both `USDT` and `USDC` as underlying tokens. but now we only used the pool asset.
<img width="959" alt="Screenshot 2024-05-21 at 9 59 35 AM" src="https://github.com/clearpool-finance/yield-server/assets/50677754/69396977-2e89-41f5-bfcd-dd629eb2d84f">

Reply:
@Addis-Belete I mean the `chain` name

Reply:
https://github.com/clearpool-finance/yield-server/blob/master/src/adaptors/utils.js